src/odemis/driver/andorcam2.py

```diff
@@ -603,7 +603,15 @@ def __init__(self, name, role, device=None, emgains=None, shutter_times=None,
             # recommended one, but higher speeds can be used, given that the voltage
             # is increased. The drawback of higher clock voltage is that it can
             # introduce extra noise.
-            vror_choices = self._getVerticalReadoutRates()
+            try:
+                vror_choices = self._getVerticalReadoutRates()
+            except AndorV2Error as ex:
+                # Some cameras report SETFUNCTION_VREADOUT but don't actually support it (as of SDK 2.100)
+                if ex.errno == 20991:  # DRV_NOT_SUPPORTED
+                    logging.debug("VSSpeed cannot be set, will provide control for it")
+                    vror_choices = {None}
+                else:
+                    raise
```

Comment:
shouldn't there be an exception argument after raise?

Reply:
This is a special Python syntax which means "pass on the exception in which you currently are".